r/geoarrow/tests/testthat/test-wk-compat.R

```diff
@@ -105,3 +105,51 @@ test_that("as_geoarrow_array() for wk generates the correct metadata", {
     sprintf('{"edges":"spherical"}')
   )
 })
+
```

Comment:
Relates to https://github.com/geoarrow/geoarrow-c/pull/79#discussion_r1411532145 and https://github.com/geoarrow/geoarrow-c/pull/79#discussion_r1411532590, add some geoedesic and crs checks.